README.md

```diff
@@ -54,3 +55,6 @@ License
 -------
 
 Akka is Open Source and available under the Apache 2 License.
+
+[akka-dependency-badge]: https://index.scala-lang.org/count.svg?q=dependencies%3Aakka%2Fakka-actor+OR+dependencies%3Aakka%2Fakka-stream+OR+dependencies%3Aakka%2Fakka-http-experimental&subject=scaladex:&color=blue&style=flat-square "Built with Akka"
```

Comment:
if you want to add it to the other badges it's part of the http://shields.io/ api